src/Workspaces/Remote/ServiceHub/Services/SnapshotService.JsonRpcAssetSource.cs

```diff
@@ -50,6 +50,25 @@ public override async Task<IList<(Checksum, object)>> RequestAssetsAsync(int sco
                 }
             }
 
+            public override async Task<bool> IsExperimentEnabledAsync(string experimentName, CancellationToken cancellationToken)
+            {
+                using (RoslynLogger.LogBlock(FunctionId.SnapshotService_IsExperimentEnabledAsync, experimentName, cancellationToken))
+                {
+                    try
+                    {
```

Comment:
this. I think you just need
return _owner.RunServicveAsync …. but doesn't need outter try/catch. the other one has it so that it Fault the OOP process. but this one shouldn't fault OOP. instead we will show InfoBar to users.

Reply:
@heejaechang Confirm I did this right? Also added a comment at the other place to explain why it's just there.

Reply:
yep